src/utils/swaggerOptions.js

```diff
@@ -19,12 +19,12 @@ const options = {
     },
     servers: [
       {
-        url: 'http://localhost:3000/api'
+        url: 'http://localhost:3000/api/auth'
```

Comment:
Oh! You've added the limitation to swagger documentation. This means swagger will only load the documentation from the `auth` route.
Instead, use `http://localhost:3000/api`

Reply:
Changing it right away.